File: src/main/resources/public/js/models/book.js
(function() {
    "use strict";

    if(window.bookcase === undefined) {
        window.bookcase = {};
    }

    bookcase.Book = Backbone.Model.extend({
        uuid: '',
        isbn10: '',
        isbn13: '',
        title: '',
        authors: [],
        pages: 0
    });

})();

File: library-dto/src/main/java/com/espressoprogrammer/library/dto/Book.kt
package com.espressoprogrammer.library.dto

data class Book(val uuid: String? = null,
                val isbn10: String? = null,
                val isbn13: String? = null,
                val title: String? = null,
                val authors: List<String> = emptyList(),
                val image: String? = null,
                val pages: Int? = null)